### PR TITLE
doc(rome_cli): Use permalinks in `rome init`

### PR DESCRIPTION
**Pull request by MichaReiser**
## Summary

This PR sets up static permalinks for the URLs used by the CLI so that changes to the website structure no longer require updating the CLI. 

This PR also slightly improves the rendering of the `rome init` command.

## Test Plan

* I opened the permalinks from my browser and verified that they redirect me to the correct sections

Output of rome init

### CODE REVIEW

**Review by xunilrj — APPROVED** [review]

**Comment by xunilrj** [comment]
Wondering if we should support OSC8 for links.  
https://gist.github.com/egmontkob/eb114294efbcd5adb1944c9f3cb5feda#the-escape-sequence

**Comment by leops** [comment]
> Wondering if we should support OSC8 for links. https://gist.github.com/egmontkob/eb114294efbcd5adb1944c9f3cb5feda#the-escape-sequence

That's actually already supported through the `Hyperlink` markup element, but I guess it may not be needed here if we rely on the terminal client automatically detecting links and making them clickable ?

**Comment by MichaReiser** [comment]
I changed it to use Hyperlink instead. The downside is that the links are more difficult to spot

grafik

File: crates/rome_cli/tests/snapshots/main_init/creates_config_file.snap [hunk 1]
```diff
@@ -19,18 +18,26 @@ expression: content
...
-	2. Try a command
-		rome ci is used to validate your code, verify formatting, and check for lint errors. Run rome --help for a full list of commands and flags
-	3. Read the documentation
-		Our website serves as a comprehensive source of guides and documentation: https://rome.tools/docs
-	4. Get involved in the community
-		Ask questions, get support, or contribute by participating on GitHub (https://github.com/rome/tools), or our community Discord (https://discord.gg/rome)
+
+Welcome to Rome! Let's get you started...
+
+Files created ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
```
Comment (ematipico) [inline crates/rome_cli/tests/snapshots/main_init/creates_config_file.snap:24]: nit: that is a long line... 😆 imagine it on terminals that have their window reduced. Should we use the minimum length that we have for the new diagnostics?

**Review by ematipico — APPROVED** [review]

File: crates/rome_cli/tests/snapshots/main_init/creates_config_file.snap (hunk 1 above)
Comment (MichaReiser) [reply crates/rome_cli/tests/snapshots/main_init/creates_config_file.snap:24]: I reduced the line length but made sure it is still as long as the longest text. 

Ideally, the line would use the terminal width to determine the ideal width but this isn't supported by our markup today.

**Comment by xunilrj** [comment]
> I changed it to use Hyperlink instead. The downside is that the links are more difficult to spot

You probably want this to make it more readable, right?

```rs
 <Hyperlink href="https://rome.tools/docs">"https://rome.tools/docs"</Hyperlink>
```

**Comment by MichaReiser** [comment]
> > I changed it to use Hyperlink instead. The downside is that the links are more difficult to spot
> 
> You probably want this to make it more readable, right?
> 
> ```rust
>  <Hyperlink href="https://rome.tools/docs">"https://rome.tools/docs"</Hyperlink>
> ```

How's that different to what the CLI does by default? Overall,  having the inline links makes the message easier to read.

**Comment by xunilrj** [comment]
> How's that different to what the CLI does by default? Overall, having the inline links makes the message easier to read.

It will make the URL clickable if the terminal supports the extension and do not search for urls.

**Pull request merged**